Samples/VSTO-shared-code-migration/README.md

```diff
@@ -0,0 +1,158 @@
+---
+page_type: sample
+products:
+- office-excel
+- office-365
+languages:
+- javascript
+extensions:
+  contentType: samples
+  technologies:
+  - Add-ins
+  createdDate: 12/2/2019 1:25:00 PM
+description: ""
+---
+
+# Use a shared library to migrate your Visual Studio Tools for Office add-in to an Office web add-in
+
+Visual Studio Tools for Office (VSTO) add-ins can only run in Office on Windows. By migrating your code to an Office web add-in, you can expand the reach of your add-in to mobile and online platforms. Migrating the code will involve rewriting it in the JavaScript language, and also updating it to use the Office.js APIs. However, you can save on migration cost by creating a shared library and keeping some of your existing add-in code in its current language.
+
+This sample includes a start folder that accompanies the article TBD... If you want to learn how to go through the steps over creating a shared library for an existing VSTO add-in, follow this article.
+
+And the sample contains a complete folder that contains the completed solution. If you want to explore each section of the completed solution, follow the steps in this readme to learn more.
+
+The sample starts with an Excel VSTO add-in that provides functionality for the user to analyze a cell of text. They can view the unicode values of the text characters. The logic to convert the text to unicode is what will be shared to the web add-in.
+
+![Overview diagram of Excel custom function calling an Azure function that returns a result.](images/azure-function-overview.png)
+
+## Applies to
+
+-  Excel, Outlook, PowerPoint, Word on Windows, Mac, and in a browser.
```

Comment:
Maybe this could be restructured to make it clearer it's not just Word that's cross-platform.

Reply:
fixed